src/isotools/_transcriptome_io.py

```diff
@@ -644,132 +647,132 @@ def _check_chimeric(chimeric):
     return chimeric_dict, non_chimeric
 
 
-def _add_sample_gene(t, g_start, g_end, g_infos, tr_list, chrom, novel_prefix):
+def _add_sample_gene(t, gene_start, gene_end, gene_infos, transcript_list, chrom, novel_prefix):
```

Comment:
`t` should be changed to `transcriptome` (if I understand correctly) in this function, as well as in `_add_novel_genes` and in `_add_chimeric`